gard_owl_ingest/mondo_mapping_status.py

```diff
@@ -127,49 +117,62 @@ def gard_mondo_mapping_status():
         gard_sssom_proxy_df, mondo_sssom_df, how='left', left_on='object_id', right_on='mondo_object_id').fillna('')\
         .sort_values(by='subject_id')
     del gard_sssom_proxy_df2['mondo_object_id']
+    gard_sssom_proxy_df2['mapping_justification'] = 'semapv:ManualMappingCuration'
     gard_sssom_proxy_df2['gard_mondo_mapping_type'] = gard_sssom_proxy_df2.apply(
         lambda x: 'proxy_existing' if x.subject_id in existing_gard_mondo_mappings
         else 'proxy_new' if x.mondo_id != '' else 'unmapped_proxy_or_direct', axis=1)
     # - Filter: proxy_existing: Because proxy info not needed for existing mappings to be concatenated
     gard_sssom_proxy_df3 = gard_sssom_proxy_df2[gard_sssom_proxy_df2['gard_mondo_mapping_type'] != 'proxy_existing']
-    # - Add existing mappings
+    # - Add existing mappingsd
     rows = []
-    for row in gard_mondo_sssom_df.itertuples():
+    for row in gard_mondo_sssom_from_mondo_df.itertuples():
+        # noinspection PyUnresolvedReferences pycharm_doesnt_recognize_tuple_props
         rows.append({
             'subject_id': row.gard_id,
-            'predicate_id': '',
-            'object_id': '',
+            'predicate_id': row.predicate_id,
+            'object_id': row.mondo_id,
             'mondo_id': row.mondo_id,
             'mondo_label': row.mondo_label,
+            'mapping_justification': row.mapping_justification,
             'gard_mondo_mapping_type': 'direct_existing',
         })
     gard_sssom_direct_df = pd.DataFrame(rows)
     # todo: ideally, would sort proxy_new -> unmapped... -> direct_existing
     # todo: ideally would have last 3 col order: gard_mondo_mapping_type, mondo_id, mondo_label
-    gard_sssom_df = pd.concat([gard_sssom_proxy_df3, gard_sssom_direct_df])\
-        .sort_values(['gard_mondo_mapping_type', 'subject_id', 'predicate_id', 'object_id', 'mondo_id', 'mondo_label'])
-    gard_sssom_df.to_csv(OUT_DIR / 'gard-mondo.sssom-like.tsv', index=False, sep='\t')
-
-    # Get a list of obsolete GARD terms that are still in Mondo
+    gard_sssom_like_df = pd.concat([gard_sssom_proxy_df3, gard_sssom_direct_df])\
+        .sort_values(['gard_mondo_mapping_type', 'subject_id', 'predicate_id', 'object_id', 'mondo_id', 'mondo_label',
+                      'mapping_justification']).fillna('')
+    gard_sssom_like_df.to_csv(ANALYSIS_OUTDIR / 'gard-mondo.sssom-like.tsv', index=False, sep='\t')
+
+    # gard-mondo.sssom.tsv
+    gard_sssom_df = gard_sssom_like_df[gard_sssom_like_df['gard_mondo_mapping_type'] != 'unmapped_proxy_or_direct']
+    gard_sssom_df = gard_sssom_df[[col for col in gard_sssom_df.columns if col != 'gard_mondo_mapping_type']]
+    # TODO: #1 temp columns to help with Mondo mapping deduping. remove when done
+    gard_sssom_df['temp_original_object_id'] = gard_sssom_df['object_id']
```

Comment:
This column should be deleted from the final mapping table!

Reply:
I know we talked about this on meeting, but just following up here. I put the `temp_` in the column title and added a `todo` above it with a hashtag. It's in my final cleanup steps for this task to remove this, so no worries!